scripts/install_from_src_ubuntu

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euo pipefail
+
+clean_slate()
+{
+	echo "                      =============================================="
+	echo "                      If you are within a conda env please deactivate it and come back"
```

Comment:
Refer to comment in `install_from_source_mac`.

Also in EC2 environment DL AMI is the most preferred environment which comes in with conda for package/environment management.